README.markdown

```diff
@@ -29,6 +29,11 @@ If your CA is expiring soon (or it's already expired and Puppet has stopped work
 * Generate a new CA certificate using the existing CA keypair.
 * Distribute the new CA cert to every node in your Puppet infrastructure.
 
+If you are using a CRL with Puppet, the CRL may also expire at the same time as the CA certificate. While the CRL expiration is normally updated whenever a certificate is revoked and thus is unlikely to expire, in rare cases where certificate revocation is not done the CRL may expire. When your CA certificate is regenerated, you should:
+
+* Update the CRL with a new expiration. This will happen automatically when the CA certificate is regenerated.
+* Distribute the updated CRL to every node in your Puppet infrastructure.
+
```

Comment:
@adrienthebo Since we're rolling this into the primary workflow, I don't think we need to present these as separate tasks. (Also, isn't everyone using a CRL whether they know it or not?) 

I suggest this:

> If your CA is expiring soon (or it's already expired and Puppet has stopped working), you should:
> 
> * Generate a new CA certificate using the existing CA keypair. This will also automatically update the expiration date of the certificate revocation list (CRL).
> * Distribute the new CA cert and CRL to every node in your Puppet infrastructure.

That changes the messaging from "you have a decision to make about CRLs" to "we'll update any CA-related things that might need updating."

Then, I would move your explanatory text down below to the section about the crl action, where it fits better and won't distract people who don't need to care about it.